app/jobs/process_entry_job.rb

```diff
@@ -13,18 +13,20 @@ def perform(entry)
   end
 
   def self.model_integrations_for(entry)
-    model = entry.model
+    self.ensure_integrations_for(entry)
 
-    integrations = Integration.retry_record_not_unique do
-      case model.record
-      when Proxy then PROXY_INTEGRATIONS.map { |i| i.new(entry) }.each(&:call)
-      when Provider then CreateK8SIntegration.new(entry).call
-      end
+    model = entry.model
+    integrations = Integration.for_model(model)
+    integrations.each.with_object(model)
+  end
 
-      Integration.for_model(model)
+  def self.ensure_integrations_for(entry)
+    case entry.model.record
+    when Proxy
+      self.const_get(:PROXY_INTEGRATIONS).map { |integration| integration.new(entry) }.each(&:call)
```

Comment:
Why this `const_get` ?

Reply:
I wanted to override it in the test
https://github.com/3scale/zync/blob/347ee9a2e9afc4dda6af95675564b8f9c5a22758/test/jobs/process_entry_job_test.rb#L70

Reply:
Better to use class_attribute then?